src/comet_llm/experiment_api/request_exception_wrapper.py

```diff
@@ -39,6 +43,14 @@ def wrapper(*args, **kwargs) -> Any:  # type: ignore
                             f"{comet_url}. Check that your Comet "
                             f"installation is up-to-date and check the traceback for more details."
                         )
+                if exception.response is not None:
```

Comment:
I think it's not the last `sdk_error_code` handling we'll have since this wraps every low-level backend call we have. And we definitely don't want to add a new if-block whenever we have a new error code.

Let's add an abstraction here that will handle the Response instance. Here it might be used for example as a function `handle_response(exception.response)` that returns `exception_args` for `CometLLMException`.

Should be convenient to use a dictionary-based registry here to store different handlers for different sdk_error_codes.